cell-based-architecture-for-agents/scripts/create-secrets.sh

```
@@ -0,0 +1,117 @@
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ENV_FILE="${SCRIPT_DIR}/.env"
+
+if [[ ! -f "${ENV_FILE}" ]]; then
+  echo "error: ${ENV_FILE} not found. Copy scripts/.env.example to scripts/.env and fill it in." >&2
+  exit 1
+fi
+
+# shellcheck disable=SC1090
+set -a; source "${ENV_FILE}"; set +a
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)